discord-bot/deploy.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+SERVICE_NAME="manifold-discord-bot"
+
+ENV=${1:-dev}
+case $ENV in
+    dev)
+      ENVIRONMENT=DEV
+      GCLOUD_CPU=1
+      GCLOUD_PROJECT=dev-mantic-markets ;;
+    prod)
+      ENVIRONMENT=PROD
+      GCLOUD_CPU=4
+      GCLOUD_PROJECT=mantic-markets ;;
```

Comment:
You should just stick to 1 CPU. 4 CPUs was because the replicator actually has a lot of work to do.